analysis/estimate/run-estimate-analysis.sh

```diff
@@ -12,16 +12,25 @@ cd "$basedir"
 data_dir="data"
 script_dir="scripts"
 scpca_dir="${data_dir}/scpca"
-tpm_dir="${data_dir}/tpm"
-pseudobulk_dir="${data_dir}/pseudobulk"
 
 mkdir -p $scpca_dir
-mkdir -p $tpm_dir
-mkdir -p $pseudobulk_dir
 
 ensembl_symbol_map_file="${data_dir}/ensembl-symbol-map.tsv"
 
 # Sync data files from S3
 Rscript ${script_dir}/sync-data-files.R \
   --output_dir "${scpca_dir}" \
   --ensembl_symbol_map_file "${ensembl_symbol_map_file}"
+
+for project_dir in $scpca_dir/*; do
+    project_id=$(basename $project_dir)
+    
+    expression_file="${data_dir}/${project_id}_expression.rds"
+    project_dir="${scpca_dir}/${project_id}"
```

Comment:
Do you need to define this if you are already defining it above in the loop?

Reply:
sure don't!!